code-challenges.js

```diff
@@ -22,22 +22,58 @@ const temp2 = 350
 const temp3 = 212
 
 
+
+
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+//console.log(myNumbers1.concat(myNumbers2).length)
+//log and combine myNumbers1 & myNumbers2--> 
+//output -->[[3, 7, 0, 36, -9, 8, -7, 42, 9, 13
+//.length counts the number of indexes -->
+//prints the length of the array instead of the output above making
+//output -->10
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ahplA"
 
 const currentCohort = "Alpha 2022"
 
+//console.log(currentCohort.split("").reverse().join(""));
+
+//logs value of currentCohort >
+//"splits" the values of the string into a index of characters
+//"reverses" the order of the string index>
+//"joins" them back together into a reversed string>
+//output-->"1202 ahplA"
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
 const myArray = [13, 34, 5, 10, 27, 42]
 
 
+
+for(let i = 0; i < myArray.length; i++){
+    if(myArray.indexOf(i) % 2 !== 0){
+      console.log("odd")
+    } else {
+      console.log("even")
+    }
+  }
+
+// currently working the way i want to but not getting the correct math
+//let i equal zero and is 0
+//if i is greater than myArray and its length
+//increment the value of i
+//i returns the original value
+//----> output is wrong but prints (odd,odd,odd,odd,odd,even)
```

Comment:
Great job in writing out your thought process!  indexOf locates the specific index number of a parameter that is passed into it.  Instead of locating the index of i, we can use i as our index number placeholder in the for loop to access the individual values inside the array.

Reply:
thank you i have figured out how to fix my process so hopefully i will not run into these errors anymore